code/config_handler.py

```diff
@@ -0,0 +1,49 @@
+from configparser import ConfigParser
+
+import pyodbc
+
+
+def load_config():
+    # Read configuration file
+    parser = ConfigParser()
+    parser.read('../dev.ini')
+    return parser
+
+
+def get_SQLCONFIG():
+    parser = load_config()
+    # Read corresponding file parameters
+    _driver = parser.get("db", "driver")
```

Comment:
Why do you prepend local variables with underscore?

Reply:
updated